audiorecorder

```diff
@@ -315,8 +315,8 @@ fi
 
 # Passthrough mode
 if [ "${runtype}" = "passthrough" ] ; then
-    rec -r "${SAMPLE_RATE_NUMERIC}" -b 32 -L -e signed-integer --buffer 500000 -p remix 1 2 | ffmpeg -i - -f wav -c:a pcm_s16le -ar 44100 -filter_complex "${CHANNELS}" -map "${LAYOUT}" - |\
-    ffplay -window_title "Current Settings Will Record Channel(s): ${channel_layout}" -f lavfi \
+    rec -r "${SAMPLE_RATE_NUMERIC}" -b 32 -L -e signed-integer --buffer 500000 -p remix ${SOXCHANNELS} | ffmpeg ${CHANNEL_LAYOUT} -i - -f wav -c:a pcm_s16le -ar 44100 - |\
+    ffplay -window_title "Monitoring Channel(s): ${channel_layout}" -f lavfi \
     "amovie='pipe\:0',${FILTER_CHAIN}" 2> /dev/null
```

Comment:
I suggest to indent the
```
"amovie='pipe\:0',${FILTER_CHAIN}" 2> /dev/null
```
line, because it’s not another pipe, but the same ffmpeg command. To me it’s easier to read, but _de gustibus non est disputandum_.